python/private/repo_utils.bzl

```diff
@@ -92,6 +82,7 @@ def _logger(ctx, name = None):
         debug = lambda message_cb: _log(2, "DEBUG", message_cb),
         info = lambda message_cb: _log(1, "INFO", message_cb),
         warn = lambda message_cb: _log(0, "WARNING", message_cb),
+        fail = lambda message_cb: _log(-1, "FAIL", message_cb, fail),
```

Comment:
The only annoying thing about the fail method is that one has to do a return in the code afterwards to satisfy buildifier.

Reply:
oh? I didn't see that buildifier error pop up.